app/main.py

```diff
@@ -2,0 +1,53 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
The `skills` variable should be an attribute of the instance, not the class. It's better to set it in the `__init__` method of the `SoftwareEngineer` class. This way, each instance of the class will have its own `skills` list. Here, it seems like you are overriding the `skills` attribute of the parent class in each child class. This could lead to unexpected behavior.